app/components/referees_review_component.html.erb

```diff
@@ -6,7 +6,10 @@
       </h3>
       <% if @editable %>
         <div class="app-summary-card__actions">
-          <%= link_to t('application_form.referees.delete'), candidate_interface_destroy_referee_path(referee.id), class: 'govuk-link' %>
+          <%= link_to candidate_interface_destroy_referee_path(referee.id), class: 'govuk-link' do %>
```

Comment:
Use `govuk_link_to`?

Reply:
Does not support blocks e.g. `do...end` currently, this is ok for now!